src/overlays/actors/ovl_Bg_F40_Swlift/z_bg_f40_swlift.h

```diff
@@ -1,13 +1,16 @@
 #ifndef Z_BG_F40_SWLIFT_H
 #define Z_BG_F40_SWLIFT_H
 
+#define BG_F40_SWLIFT_GET_PARAMS(thisx) ((thisx)->dyna.actor.params & 0xFF)
+#define BG_F40_GET_SWITCHFLAG(thisx) (((thisx)->dyna.actor.params >> 8) & 0xFF)
```

Comment:
```suggestion
#define BG_F40_SWLIFT_GET_PARAMS(thisx) ((thisx)->params & 0xFF)
#define BG_F40_GET_SWITCHFLAG(thisx) (((thisx)->params >> 8) & 0xFF)
```
Should act like the actor struct so directly to `params`

Reply:
This does not work. Should I put a cast where it's used in the actor file?

Reply:
Not a cast. Instead you would use `&this->actor`. For example
`index = BG_F40_GET_SWITCHFLAG(&this->actor);`